.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+# Variables used in this pipeline are defined in `shared-pipeline-vars`, which is `source`'d before calling `buidkite-agent pipeline upload`
+
+steps:
+  - group: Linters
+    steps:
+      - label: 🕵️ Lint & Format Check
+        key: quality-check
+        plugins: [$NVM_PLUGIN]
+        command: |
+          echo "--- 🛠 Lint & Format Check"
+          npm ci
+          make quality-check
+        notify:
+          - github_commit_status:
+              context: Lint
```

Comment:
Maybe use a less generic name for the GitHub status if we might introduce more types of linting in the future? 🤔